src/languages/elm.ts

```diff
@@ -0,0 +1,55 @@
+import { SimpleScopeTypeType } from "../typings/targetDescriptor.types";
+import { NodeMatcherAlternative } from "../typings/Types";
+import { createPatternMatchers } from "../util/nodeMatchers";
+
+const nodeMatchers: Partial<
+  Record<SimpleScopeTypeType, NodeMatcherAlternative>
+> = {
+  map: ["record_expr", "record_pattern"],
+  list: ["list_expr", "list_pattern"],
+  statement: [
+    // not all of these are "statements" in Elm, but they fit the usage pattern in Cursorless
+    "let_in_expr",
+    "case_of_expr",
+    "port_annotation",
+    "infix_declaration",
+    "if_else_expr",
+  ],
+  string: "string_constant_expr",
+  collectionItem: ["exposed_value"],
```

Comment:
Fwiw for comma-separated lists, if you just don't support them you'll get them for free.  You only need to implement "item" for cases where either of the following is true:

- The list is not surrounded by a paired delimiter such as `[]`, `()` etc, or
- The list is not comma-separated

If the list is comma-separated and surrounded by delimiters, it's supported automatically via textual magic

I'd still leave your tests, but try removing this line and seeing if they still pass

```suggestion
```